src/ai/backend/manager/models/gql_models/vfolder.py

```diff
@@ -566,14 +565,17 @@ def parse_model(
         )
 
     @classmethod
-    async def from_row(cls, graph_ctx: GraphQueryContext, vfolder_row: VFolderRow) -> Self | None:
+    async def from_row(
+        cls, graph_ctx: GraphQueryContext, vfolder_row: VFolderRow
+    ) -> Optional[Self]:
         try:
             return await cls.parse_row(graph_ctx, vfolder_row)
-        except Exception as e:
-            if isinstance(e, ModelCardProcessError):
-                error_msg = e.msg
-            else:
-                error_msg = "Unknown error"
+        except ModelCardParseError as e:
+            log.exception(
+                "Failed to parse model card from vfolder (id: {}, error: {})",
+                vfolder_row.id,
+                repr(e),
+            )
             if (
```

Comment:
Isn't the problem that an unknown error appears when an error other than ModelCardParseError occurs?
It seems that partial failure will not be possible if other Exceptions occur.